x-pack/plugins/infra/public/common/alerts/types.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { BoolQuery, Filter } from '@kbn/es-query';
+import { ALERT_STATUS_ACTIVE, ALERT_STATUS_RECOVERED } from '@kbn/rule-data-utils';
+import { ALERT_STATUS_ALL } from './constants';
+
+export type AlertStatus =
```

Comment:
You can get these types from Observability:

`import type { AlertStatus, AlertStatusFilter } from '@kbn/observability/common'`

Reply:
The **`AlertStatusFilter`** type is different type here we have:
```
export interface AlertStatusFilter {
  status: AlertStatus;
  query?: Filter['query'];
  label: string;
}
```
(So **query** has different type)
 I can still use `AlertStatus` [done]